concepticondata/conceptlists.tsv

```diff
@@ -334,3 +334,4 @@ PharaoHansen-2020-100	Pharao Hansen, Magnus	2020		100	basic, areal	Spanish	South
 Miller-1984-100	Miller, Wick R.	1984		100	basic, areal	Spanish	Uto-Aztecan languages		Miller1984	Miller1984	The current list represents a modified version of [Swadesh-1955-100](:ref:Swadesh-1955-100) with twelve substitutions. The pronouns and demonstratives found in Swadesh-1955-100 were omitted and replaced with the concepts denoting simple actions (e.g. SING, VOMIT) and natural phenomena (e.g., WIND, SNOW, SKY). The resulted list was used in the reconstruction of the Uto-Aztecan language tree based on the investigation of cognate sets in the selected thirty-two languages.	9-12	
 Mountain-1978-267	Mountain, Kathy	1978		267	areal	Spanish	Tucanoan languages		Mountain1978		This list compares 267 concepts across 12 different Tucanoan languages of South America. The concepts were taken from a 375-item list containing concepts attributed to both Swadesh and Rowe; the original concept numbers of that foundational list are indicated in the respective column here. Unfortunately, it is not clear to us by now which publication by Rowe presents a concept list. We are currently looking into this and hope to be able to update it soon. We are happy for any hints, which can be provided in the form of an issue on our GitHub repository.	10-53	
 Loukotka-1954-45	Loukotka, Čestmír	1954		45	basic, areal	Spanish, English	South American languages		Rowe1954	Rowe1954	This 45-item list was used in a comparative study of South American languages and served as the basis for their preliminary classification ([Loukotka 1944](:bib:Loukotka1944)). Since the list is not readily available as it was not officially provided by the author, it was retrieved from [Rowe 1954](:bib:Rowe1954). Moreover, the study contains a lot of unpublished material, and Loukotka gives no discussion of the evidence for the classification which is claimed to be based on a rather rudimentary method. What also needs to be mentioned is the discrepancy between the Spanish gloss *tigre* and its English translation *wildcat*, added by the author himself. We suspect that the reference was made in relation to the so-called «big cat», the more general term of wild cat species belonging to the genus Panthera. Interestingly, the same peculiar discrepancy could also be found in [Huber-1992-375](:ref:Huber-1992-375), which seems to confirm our assumption.	24	
+Chacon-2019-220	Chacon, Thiago Costa and Gonçalves, Artur Garcia and da Silva, Lucas Ferreira	2019		220	basic, areal	Portuguese	Arawakan languages	https://doi.org/10.15446/fyf.v32n2.80814	Chacon2019		A 243-item list (note that the list actually contains 243 items in contrast to the 220 stated in the paper) used as the basis for a comparative and dialectogical analysis of Arawak languages and its varieties from the Upper Rio Negro in Brazil. The data are based on documents created by missionaries in the 1950s. 		
```

Comment:
I vaguely remeber some lists stating they have XX concepts and then their data actually showed a handful less. This is the opposite case, and I'm not sure how it is usually handled, but shouldn't the list still be named after the number of concepts present? Or perhaps the number in the ITEMS column could reflect the actual value.

Reply:
This (that is, the naming of the list) is also based on the discussions in #807. I agree with the `ITEMS` column, will fix.